events/event/models.py

```diff
@@ -1,14 +1,17 @@
 from django.db import models
 from registration.models import User
+from django.utils.timezone import now
 from django.db.models.fields.related import ManyToManyField
 
-
 class Event(models.Model):
     title = models.CharField(max_length=200)
     start_date = models.DateTimeField(blank=True, null=True)
     end_date = models.DateTimeField(blank=True, null=True)
     location = models.CharField(max_length=200, null=True)
     description = models.TextField(blank=True, null=True)
+    address = models.CharField(max_length=500, null=True)
+    created_date = models.DateTimeField(blank=True, null=True)
```

Comment:
Add attribute auto_now_add=True. It makes to possible automatically create the date of the event.